devtools-ci-index.yaml

```diff
@@ -331,8 +331,8 @@
       - github-pull-request:
           status-context: "ci.centos.org PR build (fabric8-planner)"
           org-list:
-            - fabric8-planner
-          success-comment: "$ghprbPullAuthorLoginMention Your image is available in the registry: Run `docker run -e PROXY_PASS_URL=\"https://api.free-stg.openshift.com\" -p 8080:8080 {image_name}`."
+            - fabric8-ui
+          success-comment: "$ghprbPullAuthorLoginMention Your image is available in the registry. Run `docker run -p 5000:8080 {image_name}:SNAPSHOT-PR-$ghprbPullId` and visit `http://localhost:5000` to access it."
```

Comment:
We will need `registry.devshift.net/` as well in the image path, right?

Reply:
Nice catch. I missed that.